src/machine/board_stm32.go

```diff
@@ -1,4 +1,4 @@
-// +build bluepill stm32f4disco
+// +build bluepill stm32f103rb_nucleo stm32f4disco
```

Comment:
Same comment about naming the target `nucleo-f103rb` so it matches the name on the datasheet.